Extensions/P2P/JsExtension.js

```diff
@@ -154,6 +154,30 @@ module.exports = {
       .addIncludeFile('Extensions/P2P/B_p2ptools.js')
       .setFunctionName('gdjs.evtTools.p2p.useCustomBrokerServer');
 
+    extension
+      .addAction(
+        'UseOwnICEServer',
+        _('Use a custom ICE server'),
+        _(
+          'Disables the default ICE (STUN or TURN) servers list and use one of your own. ' +
+            'Note that it is recommended to add at least 1 self-hosted STUN and TURN server ' +
+            'for games that are not over LAN but over the internet. ' +
+            'This action is cumulable, you can use it multiple times to add multiple servers. ' +
```

Comment:
cumulable is not english I think?
```suggestion
            'This action cane be used multiple times to add multiple servers. ' +
```

Reply:
```suggestion
            'This action can be used multiple times to add multiple servers. ' +
```
You've made a typo too 👀